src/Adapter/Currency/CommandHandler/AbstractCurrencyHandler.php

```diff
@@ -68,19 +68,27 @@ abstract class AbstractCurrencyHandler extends AbstractObjectModelHandler
      */
     protected $validator;
 
+    /**
+     * @var PatternTransformer
+     */
+    protected $patternTransformer;
+
     /**
      * @param LocaleRepository $localeRepoCLDR
-     * @param LanguageInterface[] $languages
+     * @param array $languages
```

Comment:
```suggestion
     * @param LanguageInterface[] $languages
```

The initial PHPDoc is more accurate and matches the class field

Reply:
Idk how I changed it, probably some automatic stuff in phpstorm when i added argument